charts/prometheus/templates/cm.yaml

```
@@ -2,9 +2,9 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-{{- with .Values.server.configMapAnnotations }}
+{{- with .Values.server }}
```

I'd suggest to keep the original line:
```suggestion
{{- with .Values.server.configMapAnnotations }}
```